index.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 import promise from 'bluebird';
 import restify from 'restify';
 import corsMiddleware from 'restify-cors-middleware';
-import Qweb3 from 'qweb3';
+import { Qweb3 } from 'qweb3';
```

Comment:
The same as bodhi-graphql comments. Qweb3 import should be 
import Qweb3 from 'qweb3';

Reply:
fixed